indico/modules/events/abstracts/operations.py

```diff
@@ -66,8 +66,9 @@ def delete_abstract(abstract, delete_contrib=False):
                            'Abstract "{}" has been deleted'.format(abstract.title), session.user)
 
 
-def judge_abstract(abstract, abstract_data, judgment, contrib_session=None, merge_persons=False,
+def judge_abstract(abstract, abstract_data, judgment, judge, contrib_session=None, merge_persons=False,
                    send_notification=False):
+    abstract.judge = judge
```

Comment:
also set judgment_dt to the current time (otherwise it fails in the check constraint)